packages/main/src/features/InputSuggestions.js

```diff
@@ -318,7 +319,7 @@ class Suggestions {
 	}
 
 	_getItems() {
-		return [...this.responsivePopover.querySelectorAll("ui5-list>*")];
+		return [...this.responsivePopover.querySelectorAll("ui5-li-suggestion-item, ui5-li-groupheader")];
```

Comment:
The reason I put `>*` before was that with the new hooks, people could create their own implementation with f.e. `ui5-li-custom`. At least add it here, if it is not possible to reliably get the children.

But before that, please try with:
```js
[...this.responsivePopover.querySelector("ui5-list").children]
```

Reply:
I see, I tried it and it works this way as well